xtask/codegen/src/generate_syntax_kinds.rs

```diff
@@ -172,6 +172,19 @@ pub fn generate_syntax_kinds(grammar: KindsSrc, language_kind: LanguageKind) ->
                 }
             }
         }
+        LanguageKind::Yaml => {
+            quote! {
+                pub const fn to_string(&self) -> Option<&'static str> {
+                    let tok = match self {
+                        #(#punctuation => #punctuation_strings,)*
+                        #(#full_keywords => #all_keyword_to_strings,)*
+                        GRAPHQL_STRING_LITERAL => "string literal",
```

Comment:
GraphQL code leaked in here :) .

Reply:
oh god, I may or may not have done a little pasting there :D Fixed it and will update soon.